File: Module_2/src/extra_exercise/crud_product/models/Product.java
package extra_exercise.crud_product.models;

public class Product {
    private int id;
    private String name;
    private String label;
    private int price;

    public Product() {
    }

    public Product(int id, String name, String label, int price) {
        this.id = id;
        this.name = name;
        this.label = label;
        this.price = price;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    @Override
    public String toString() {
        return  "Product "+this.id+
                ", Name "+this.name+
                ", Label "+this.label+
                ", Price "+this.price;
    }
}

File: Module_5/session_03/practice/product-list-bootstrap/src/app/product/model/Product.ts
export class Product {

  private _id:number;
  private _name:string;
  private _price:number;
  private _quantity:number;
  private _manufacture:string;


  constructor(id: number, name: string, price: number, quantity: number, manufacture: string) {
    this._id = id;
    this._name = name;
    this._price = price;
    this._quantity = quantity;
    this._manufacture = manufacture;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get price(): number {
    return this._price;
  }

  set price(value: number) {
    this._price = value;
  }

  get quantity(): number {
    return this._quantity;
  }

  set quantity(value: number) {
    this._quantity = value;
  }

  get manufacture(): string {
    return this._manufacture;
  }

  set manufacture(value: string) {
    this._manufacture = value;
  }
}
